packages/workspace/src/browser/workspace-service.ts

```diff
@@ -261,10 +261,18 @@ export class WorkspaceService implements FrontendApplicationContribution {
     }
 
     /**
-     * Returns `true` if there is an opened workspace in theia, and the workspace has more than one root.
+     * Returns `true` if a multiple-root workspace is currently open.
      * @returns {boolean}
      */
     get isMultiRootWorkspaceOpened(): boolean {
+        return !!this.workspace && !this.workspace.isDirectory;
```

Comment:
```ts
return this.opened && !this.workspace.isDirectory;
```
It would be aligned with the doc, you check `open` and not if the `_workspace` is defined, and with `isMultiRootWorkspaceEnabled` too.

Reply:
Can I open a multi-root workspace with `workspace.supportMultiRootWorkspace: false`?

Reply:
> ```ts
> return this.opened && !this.workspace.isDirectory;
> ```

this looks nice. however i am getting a typescript error that complains `this._workspace` can possibly be undefined.
```
[compile] src/browser/workspace-service.ts(268,32): error TS2532: Object is possibly 'undefined'.
```

I think we could either 
- leave the code as is, or 
- replace it with `return this.opened && !!this.workspace && !this.workspace.isDirectory;`

which do you prefer ?

Reply:
> Can I open a multi-root workspace with `workspace.supportMultiRootWorkspace: false`?

no you cannot (at lease `shouldn't` :)). If it happened in your environment please create a github issue and assign it to me.

Reply:
> * leave the code as is

👍